src/main/config/config.h

```diff
@@ -36,6 +36,7 @@ typedef enum {
 typedef struct pilotConfig_s {
     char craftName[MAX_NAME_LENGTH + 1];
     char pilotName[MAX_NAME_LENGTH + 1];
+    char message[8][MAX_NAME_LENGTH + 1];
```

Comment:
`#define OSD_CUSTOM_MSG_COUNT 3` somewhere, then use it in all places.
Also, allocate space for 8 messages in protocol messages, but limit it to 3 or 4 now. That way, it can be easily increased if necessary, but it does not waste memory and CPU for unused messages.